### PR TITLE
animation fixed in cards in posts page

### PR DESCRIPTION
**Pull request by mohamed-tarek-fouad**
animation fixed in cards in posts page

### CODE REVIEW

**Review by JustKira — APPROVED** [review]

File: src/components/Cards.jsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
 import axios from 'axios';
-import React, {useEffect, useState} from 'react'
+import React, { useEffect, useState, useRef } from 'react'
 import Discription from './Discription';
 import Filters from './Filters';
+import gsap from 'gsap'
```
Comment (yradwan147) [inline src/components/Cards.jsx:5]: What is gsap?

File: src/components/Discription.jsx [hunk 2]
```diff
@@ -3,8 +3,8 @@ import React from 'react'
 export default function Discription() {
```
Comment (yradwan147) [inline src/components/Discription.jsx:3]: Typo in Discription...

File: tailwind.config.js [hunk 3]
```diff
@@ -9,10 +9,26 @@ module.exports = {
             },
             fontFamily: {
                 Pop: ['Poppins', 'sans-serif']
+            },      minHeight:{
+                '3/4':'65vh',
             },
-            minHeight: {
-                '3/4': '65vh'
-            }
+            width: {
```
Comment (yradwan147) [inline tailwind.config.js:15]: What is this for?

File: src/components/Cards.jsx (hunk 1 above)
Comment (mohamed-tarek-fouad) [reply src/components/Cards.jsx:5]: librarry for animations

File: tailwind.config.js (hunk 3 above)
Comment (mohamed-tarek-fouad) [reply tailwind.config.js:15]: costmizing width for discription part

**Pull request merged**